uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-calendar-admin.controller.js

```diff
@@ -129,5 +160,16 @@ define([
         }
       };
     }
+
+    /**
+     * Shows a hint to the filtering logic
+     */
+    function showAdminFilteringHint (comment) {
+      notification.info('', [
```

Comment:
Hm... The Notification service uses CRM alerts to show notifications. Do you suggest to move the implementation of the notification to a component?

Reply:
No, I'm suggesting to add an easy hint component. The hint controller would use the notification service.

This is a mock implementation:

```js
.component('hint', {
  controller: HintController,
  controllerAs: 'hint',
  template: 'path',
  transclude: true // this will add content to the hint
});

HintController.$inject =  ['$element', 'Notification'];

function ($element, Notification) {
  var vm = this;

  vm.show = show;

  function show () {
    Notification.info($element.find('.content').html());
  }
}
```

```html
<span class="hint-component">
  <button ng-click="hint.show()">
    <i class="fa fa-info"></i>
  </button>

  <div class="content hidden" ng-transclude></div>
</span>
```

Reply:
OK, as discussed, this should be considered to be done when there is PR for staging branch.